### PR TITLE
fix: remove major field, replace with version in init command

### LINKED ISSUE #178: eik init still has incorrect version field
Still includes `major` and not `version` field when scaffolding a new eik.json file

### PR DESCRIPTION
**Pull request by digitalsadhu**
Thought I had fixed this. Major is from the early days of Eik, we don't use it any more.

Fixes #178

### CODE REVIEW

File: test/integration/init.test.js [hunk 1]
```diff
@@ -27,7 +27,7 @@ test('Initializing a new eik.json file', async t => {
     );
 
     t.equals(assets.name, '', 'eik.json "name" field should be empty');
-    t.equals(assets.major, 1, 'eik.json "major" field should equal 1');
+    t.equals(assets.version, '1.0.0', 'eik.json "major" field should equal 1');
```
Comment (stipsan) [inline test/integration/init.test.js:30]:
```suggestion
    t.equals(assets.version, '1.0.0', 'eik.json "version" field should equal 1.0.0');
```

File: test/integration/init.test.js [hunk 2]
```diff
@@ -57,8 +57,8 @@ test('Initializing a new eik.json file passing custom values', async t => {
         'eik.json "name" field should not be empty',
     );
     t.equals(
-        assets.major,
-        2,
+        assets.version,
+        '2.0.0',
         'eik.json "major" field should not be empty',
```
Comment (stipsan) [inline test/integration/init.test.js:62]:
```suggestion
        'eik.json "version" field should not be empty',
```

**Review by stipsan — CHANGES_REQUESTED** [review]
LGTM except a few leftover references to "major" 👍

**Review by stipsan — APPROVED** [review]

**Review by trygve-lie — APPROVED** [review]
lgtm :+1:

**Pull request merged**